Server/src/main/java/net/simon987/server/web/LeaderBoardPage.java

```diff
@@ -13,9 +13,12 @@
 
     @Override
     public ModelAndView handle(Request request, Response response) {
-        Map<String, Object> model = new HashMap<>(2);
+        Map<String, Object> model = new HashMap<>(5);
         model.put("session", request.session());
         model.put("stats", GameServer.INSTANCE.getUserStatsHelper().getTopNSetLength("completedVaults", 25));
+        model.put("stats", GameServer.INSTANCE.getUserStatsHelper().getTopNSetLength("deathCount", 25));
```

Comment:
Here you will have to modify the method so that it provides an array of stats with an attribute for walkDistance, deathCount and completedVaults.
Note that completedVaults itself is already an array of strings - what we're displaying is the number (`.size()` of vaults)

If you need help with that let me know

Reply:
I'm not entirely sure how to accomplish this mainly due to the variable type of model and trying to use put with GameServer.INSTANCE.getUserStatsHelper().getTopN("deathCount", 25) giving conflicting variable types

Reply:
changed